.github/workflows/release.yml

```diff
@@ -165,7 +165,7 @@ jobs:
           --repository ${{ github.repository }} \
           --commit \
           --mixed-mode-results mixed-mode-results.md \
-          ${{ steps.get_old_version.outputs.version }} ${{ steps.get_new_version.outputs.version }}
```

Comment:
I think we can get rid of the `get_old_version` step entirely, as it doesn't seem to be used any more.

Reply:
Yeah, that sounds right, I'll do that as a followup so that we can get this merged, and can do releases again.